js/index.js

```diff
@@ -80,4 +80,53 @@ for(let [i, [key, value]] of Object.entries(Object.entries(siteContent["contact"
 }
 
 let footer = document.querySelector("footer p");
-footer.innerHTML = siteContent["footer"]["copyright"];
\ No newline at end of file
+footer.innerHTML = siteContent["footer"]["copyright"];
+
+document.querySelectorAll("nav a").forEach(item => item.style.cssText = "color: green");
```

Comment:
.. item.style.cssText?!?!?! that's so cool! i had no idea! Thank you for teaching me Alejandro! :D